regression_tests/ref_counter.jl

```diff
@@ -1 +1 @@
-137
+1
```

Comment:
```suggestion
138
```

Reply:
I'd need to double check the regression test infrastructure - not sure if simply updating the `ref_counter` is enough to define a self-reference or if it retains the "last known build" (in this case 137) somehow.

Reply:
It seems pasting the output from the regression tests is sufficient (hence `1` will be a new, and sensible self-reference.